data/core.yaml

```
@@ -1832,6 +1832,8 @@ en:
         reference: "Crossing indoor features should use different levels."
       indoor-indoor_connectable:
         reference: "Crossing indoor features should be connected or use different levels."
+      power-other:
+        reference: "Power lines should not be connected to other features."
```

How about something like this, to avoid leaving the impression that power lines can’t connect to other power infrastructure like `power=tower`?

```suggestion
        reference: "Only connect power lines to features that are physically connected in the real world."
```